frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts

```diff
@@ -190,9 +190,13 @@ export class NewMeetingComponent extends BaseComponent implements OnInit {
         this.filteredOptions = this.memberFilterCtrl.valueChanges.pipe(
             startWith(''),
             map(value => {
-                const filterValue = value.toLowerCase() || '';
+                if (typeof value === 'string') {
```

Comment:
Why do we need this check?

Reply:
We need this check because this is autocomplete element, and it gives ability to make search on the selection list. That's why we can have either object (INewMeetingMember) or string (searched letter/string). 
If we don't make check and the value will object, we can have error that we can use .toLowerCase method on object.

Reply:
this is not a good way to fix this

Reply:
But why? We should check the type of the passed value and if it is not string we get field "name".